### DISCUSSION TITLE
FR: Syntax highlighting for code blocks

### DISCUSSION THREAD

**Discussion by gapmiss** [mcndt/obsidian-quickshare#2] [Ideas]
Code/pre block syntax highlighting would be a welcomed feature for myself and (I assume) other developers.

example: https://noteshare.space/note/cl6h0wmu1100201mmaukatezp#-EpcJipLRUrjttQiQ8Iu_UHypWLLez0wIjb2GLHkgzU

Very well-thought out service you've created here.

**Comment by mcndt** [comment]
Thank you for testing the plugin!

I agree that syntax highlighting is an important feature. I suspect it won't be that hard to implement.

Could you open a FR issue on the Noteshare.space repository using this link? 

https://github.com/mcndt/noteshare.space/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature+request%5D

**Comment by mcndt** [comment]
@gapmiss 

I added syntax highlighting for the 36 default languages in Highlight.js. Let me know if any languages are missing for you!

![image](https://user-images.githubusercontent.com/23149353/183262652-69e17b65-28c5-4b3e-b941-8212b9d825b0.png)

**Reply by gapmiss** [reply]
Looks great, thank you for the quick turnaround.
